scripts/release/mule/package/deb/package.sh

```diff
@@ -134,7 +135,7 @@ EOF
 
     pushd "$PKG_DIR"
 
-    STATUSFILE=build_status_${CHANNEL}_${VERSION}
+    STATUSFILE=build_status_${CHANNEL}-${ARCH_TYPE}_${VERSION}
```

Comment:
Should this be `build_status_${CHANNEL}_${ARCH_TYPE}_${VERSION}`?

Reply:
I followed the convention of the other files:
```
algorand-devtools_beta_linux-amd64_2.2.0.deb
install_beta_linux-arm_2.2.0.tar.gz
```

Reply:
NO OS_TYPE either?

Reply:
Maybe channel_os-arch_version?

Reply:
ok, i'll add that

Reply:
fixed